src/app/lib/analyticsUtils/index.js

```diff
@@ -175,22 +175,33 @@ export const getAtiUrl = (data = []) => {
   return parsedAtiValues.join('&');
 };
 
-export const getClickInfo = (elem, { service, component, label, type }) => {
-  /*
-    https://paper.dropbox.com/doc/Event-tracking--AjJpWibjeQPWsoRLFdZW13Y9Ag-3i47TvVb9IJBMJFcL8D6u
-    for click events we need to know:
-      * campaings = [service-name]-[component], e.g. yoruba-navigation
-      * creation (label) = [component]-[item], e.g. navigation-ere_idaraya
-      * creation (type) = [event-type], e.g. click or background
-      * url = the destination link
-      * format = [PAR=container-[component-name]::name~CHD=slot] = [PAR=container-navigation::name~CHD=1]
-         ^ for format, currently using the name of the component clicked as slot
-    */
-  const eventComponentInfo =
-    (elem && elem.dataset && elem.dataset.info) || 'brand-top'; // psammead components need to be updated with 'data' attrs
-  const cleanCompInfo = eventComponentInfo.split('/').join('-');
-  const format = `PAR=container-${component}::name~CHD=${cleanCompInfo.toLowerCase()}`;
-  const url = (elem && elem.href) || '/';
-
-  return `PUB-[${service}-${component}]-[=${type}]-[${label}]-[${format}]-[]-[]-[]-[${url}]`;
+export const getClickInfo = (
```

Comment:
Will this be used for other event types as well as clicks?

Reply:
I've changed this to `getEventInfo` as the props that get passed into this are named `eventProps` suggesting it may be used for other events too.